docs/source/guide/guide-observables.myst

```diff
@@ -0,0 +1,141 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Observables
+
++++
+
+The `mitiq.Observable` class is a way to represent an observable as a linear combination of Pauli strings. This class is used to compute expectation values which are mitigated by techniques in Mitiq.
+
```

Comment:
We could add a comment saying that using explicit observables is not the only way of using Mitiq. We still support the usual simplified way. E.g. something like this, but feel free to reword:
```
Note however that Mitiq can also be used without explicitly defining a `mitiq.Observable` but by implicitly introducing the observable in the body of an `executor` function that directly returns an expectation value as a `float`.
```

Reply:
After trying for a bit, I couldn't come to a clean way of adding this statement. Note that it is mentioned at the end of the guide (where to me seems more appropriate):

"If you do not provide an observable, the `executor` must compute and return the expectation value to mitigate."

I did change "This class is used to compute expectation values" to "This class *can be* used to compute expectation values" at the beginning, though.